### PR TITLE
Added VRS DirectForce Pro

### PR DESCRIPTION
**Pull request by JacKeTUs**
From https://github.com/JacKeTUs/vrs-ff

### CODE REVIEW

**Comment by JacKeTUs** [comment]
@LithiumFox, as promised, i merged multiple versions of one driver into one repository. That universal one should cover all boxes for most of wheelbases, including VRS. Please, test this with your VRS Wheelbase. If everything OK, i will archive vrs-ff and we will migrate users to this driver :smiley:

File: hid-pidff-wrapper.c [hunk 1]
```diff
@@ -23,6 +23,12 @@ static const struct hid_device_id pidff_wheel_devices[] = {
 		.driver_data = PIDFF_QUIRK_FIX_WHEEL_DIRECTION | PIDFF_QUIRK_FIX_PERIODIC_ENVELOPE },
 	{ HID_USB_DEVICE(USB_VENDOR_ID_MOZA, USB_DEVICE_ID_MOZA_R16_R21),
 		.driver_data = PIDFF_QUIRK_FIX_WHEEL_DIRECTION | PIDFF_QUIRK_FIX_PERIODIC_ENVELOPE },
+	{ HID_USB_DEVICE(USB_VENDOR_ID_MOZA, USB_DEVICE_ID_MOZA_R16_R21),
```
Comment (Lawstorant) [inline hid-pidff-wrapper.c:26]: R16_R21 duplication

**Review by Lawstorant — CHANGES_REQUESTED** [review]

**Comment by JacKeTUs** [comment]
Need to wait for https://github.com/JacKeTUs/universal-pidff/pull/12 to merge

**Comment by ccalhoun1999** [comment]
I've managed to build and test this branch on my vrs directforce pro with no issues.

**Pull request merged**